ux.symfony.com/templates/_nav.html.twig

```diff
@@ -0,0 +1,45 @@
+
+{% block menu_items %}
+    <a href="{{ path('app_turbo') }}" class="AppNav_item">Turbo</a>
+    <div class="dropdown">
+        <button
+                class="AppNav_item"
+                id="components-dropdown-button"
+                data-bs-toggle="dropdown"
+                aria-expanded="false"
+        >Packages
```

Comment:
I would either move `Packages` onto a new line or move `<twig:Icon` onto this same line.

And, there is some extra indentation on the button attributes.